mandatory/2-function-creation.js

```diff
@@ -25,7 +39,35 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
+  let pos;
+  for (let i = 0; i < arrayOfStrings.length; i++) {
+        let aString = arrayOfStrings[i].trim(); // removes any spaces in the beginning or end of each string
+        while ((pos = aString.indexOf("/")) >= 0) {
```

Comment:
there is a ``replace()`` function that you can use so you shouldn't be doing all this while loop